### PR TITLE
[SM] Reviewers workflows

### PR DESCRIPTION
**Pull request by pivotal-marcela-campo**
Outlines of Service Management working group processes.

Please leave this PR open until the Service Management working group discusses and votes.

### CODE REVIEW

File: toc/working-groups/service-management/working-agreements/2021-03-09-reviewer-workflows.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# Reviewer Workflows for Issues and PRs
+
+**Date of Forum to be Discussed:** Thursday 10 March, 2022 11 am UTC / 4:30 pm IST
+See the community calendar for more details.
+
+**Author:** Marcela Campo - @pivotal-marcela-campo
+
+## Problem
+Our working group has many different projects maintained by disjointed groups of people.
+In order to run a healthy open source community we need to make sure that we triage issues and merge PRs in a timely fashion.
```
Comment (FelisiaM) [inline toc/working-groups/service-management/working-agreements/2021-03-09-reviewer-workflows.md:10]: nitpick: I would prefer `review PRs` rather then `merge PRs` just to set the expectation that not all PRs are to be merged

**Comment by FelisiaM** [comment]
+1

**Review by anoopjb — COMMENTED** [review]
+1

**Review by rkoster — APPROVED** [review]

**Comment by beyhan** [comment]
@pivotal-marcela-campo is this still relevant and do you need approval from the TOC to merge it?

**Comment by pivotal-marcela-campo** [comment]
Thanks @beyhan for reminding me of this. I'll close this as we haven't really needed this kind of process in the group. I'll submit another PR if we need to.

**Pull request closed without merging**